src/pages/api/ministers/index.js

```diff
@@ -31,32 +31,13 @@ const api = {
   }
   */
   get: async (req, res) => {
-    let { limit = 15, page = 1 } = req.query;
-    limit = Number(limit);
-    page = Number(page);
-    const query = omit(['limit', 'page'], req.query);
-    const startRange = (page - 1) * limit;
-    const endRange = startRange + limit - 1;
-    const { data, error, count } = await supabase
+    const { data, error } = await supabase
       .from('ministers')
-      .select('*', { count: 'exact' })
-      .match(query)
-      .range(startRange, endRange);
-    if (data && isEmpty(data)) {
```

Comment:
We can get rid of the isEmpty and omit imports since they are no longer being used.